server/pkg/asset/preview_type.go

```diff
@@ -13,6 +13,7 @@ const (
 	PreviewTypeGEO     PreviewType = "GEO"
 	PreviewTypeGEO3D   PreviewType = "GEO3D"
 	PreviewTypeMODEL3D PreviewType = "MODEL3D"
+	PreviewTypeUNKNOWN PreviewType = "UNKNOWN"
```

Comment:
```suggestion
	PreviewTypeUNKNOWN PreviewType = ""
```

Zero value is suitable, I think.

Reply:
I am getting a lot of errors because of this change,
now there is come confusion between `(*PreviewType)(nil)` and `(*PreviewType)("")`
also the `PreviewTypeUnknown` is "unknown" and not "" in gql and the adapter level
so is it okay not to do this change?